spacecmd/src/spacecmd/misc.py

```diff
@@ -824,6 +824,14 @@ def load_caches(self, server, username):
         self.packages_by_id_cache_file
     )
 
+    # Fix String IDs to ints
+    # JSON structure is "id : value", e.g. [ "123": "example.com", ... ]
```

Comment:
In general it looks good for me, but just a nitpick comment. Are you sure about the example of json? Maybe `{..}` should be instead of `[...]`